doc/techref/projections.md

```diff
@@ -33,9 +33,9 @@ The table below shows the projection codes for the 31 GMT map projections:
 | **L**{{ lon0 }}/{{ lat0 }}/{{ lat1 }}/{{ lat2 }}/*width*    | {doc}`/projections/conic/conic_lambert` |
 | **M**[{{ lon0 }}/[{{ lat0 }}/]]*width*                      | {doc}`/projections/cyl/cyl_mercator` |
 | **N**[{{ lon0 }}/]*width*                                   | {doc}`/projections/misc/misc_robinson` |
-| **Oa**{{ lon0 }}/{{ lat0 }}/*azimuth*/*width*[**+v**]       | {doc}`/projections/cyl/cyl_oblique_mercator_1` |
-| **Ob**{{ lon0 }}/{{ lat0 }}/{{ lon1 }}/{{ lat1 }}/*width*[**+v**] | {doc}`/projections/cyl/cyl_oblique_mercator_2` |
-| **Oc**{{ lon0 }}/{{ lat0 }}/{{ lonp }}/{{ latp }}/*width*[**+v**] | {doc}`/projections/cyl/cyl_oblique_mercator_3` |
+| **Oa**{{ lon0 }}/{{ lat0 }}/*azimuth*/*width*[**+v**]             | Oblique Mercator projection: {doc}`1. origin and azimuth </projections/cyl/cyl_oblique_mercator>` |
+| **Ob**{{ lon0 }}/{{ lat0 }}/{{ lon1 }}/{{ lat1 }}/*width*[**+v**] | Oblique Mercator projection: {doc}`2. two points </projections/cyl/cyl_oblique_mercator>` |
+| **Oc**{{ lon0 }}/{{ lat0 }}/{{ lonp }}/{{ latp }}/*width*[**+v**] | Oblique Mercator projection: {doc}`3. origin and projection pole </projections/cyl/cyl_oblique_mercator>` |
```

Comment:
I prefer to also combine the three lines into one, since it's just one projection with different ways to specify the projection parameters.

The combined syntax is like:
```
**Oa**|**b**|**c**{{ lon0 }}/{{ lat0 }}/*azimuth*/*width*[**+v**]
```

Reply:
First I also thought about rewritting this to one line, but the three versions of the Oblique Mercator projection differ at the end:

```
-  **Oa**{{ lon0 }}/{{ lat0 }}/*azimuth*/*width*[**+v**]          
-  **Ob**{{ lon0 }}/{{ lat0 }}/{{ lon1 }}/{{ lat1 }}/*width*[**+v**] 
-  **Oc**{{ lon0 }}/{{ lat0 }}/{{ lonp }}/{{ latp }}/*width*[**+v**] 
```

Reply:
Yes, you're absolutely right about it.